src/test/java/org/jenkinsci/plugins/build_token_root/BuildRootActionTest.java

```diff
@@ -76,10 +76,10 @@
         wc.assertFails(p.getUrl() + "build?token=secret", HttpURLConnection.HTTP_FORBIDDEN);
         j.waitUntilNoActivity();
         assertEquals(0, p.getBuilds().size());
-        wc.goTo("buildByToken/build?job=" + p.getFullName() + "&token=secret&delay=0sec");
+        wc.goTo("buildByToken/build?job=" + p.getFullName() + "&token=secret&delay=0sec", null);
```

Comment:
Please strengthen the test to prove that it is sending a 201 with a reasonable-looking URL.